src/hooks/screen/useChatScreenHook.ts

```diff
@@ -246,14 +245,15 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
         }
       });
     } catch (e) {
-      if (e?.response?.data?.status === 'Channel is blocked') return;
-
-      setTimeout(async () => {
-        await sendChat(message, attachments, iteration + 1, currentChatSchema);
-      }, 1000);
+      console.log('[ERROR] error sending chat', e);
+      // if (e?.response?.data?.status === 'Channel is blocked') return;
     }
   };
 
+  const sendChatMutation = useMutation(sendChat, {
+    retry: true
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useMutation` hook from 'react-query' is used with the `sendChat` function and the `retry: true` option. This will automatically retry failed mutations.
```diff
+ const sendChatMutation = useMutation(sendChat, {
+   retry: true
+ });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->